### DISCUSSION TITLE
Cooling of the Raspberry Pi

### DISCUSSION THREAD

**Discussion by oliverk71** [mcguirepr89/BirdNET-Pi#133] [Ideas]
Since we have at least some really hot days in summer and the pi comes without any heat sinks, we should discuss appropriate cooling. The current version of raspian throttles the pi if CPU temperature rises above 60°C. I guess this can be changed and this might be an option, but even if, let's talk about cooling a pi. On a hot summer day temperature can rise up to 40°C in my area. So there are only 20 K left for the pi before it starts throttling. Without appropriate cooling, you will possibly run into trouble.

I bought aluminium cooler cases (Joy-tec and Miuzei) for my pi 3B and they have the advantage that they touch the hot spots (chips) on the pi and spread the heat away from them. It's not perfect, but it's okay. However, I think about other options, too. Including heatpipes. Heatpipes are copper tubes filled with a liquid, which evaporates on the hot spot and condenses on the other (cooler) end. They are used in most desktop CPU cooler, nowadays. I think about getting some heatpipes and combining them with a heat sink for efficient cooling. What do you think? May be overkill and some work, too. 

Another option is to buy a ready-made cooler with heat-pipes. There is for example the Geekpi Ice Tower Cooler. It comes with a fan, but I really hate fans, especially when it comes to outdoor use. But I guess even without fan it is a nice option. And of course, you can mount the fan, if you want. I also think about combining a aluminium cooler case with this heatpipe cooler. This way, the other chips can be cooled, too.

What do you think? Total overkill? Easier to dig a hole and place the pi there?

**Comment by nilspupils** [comment]
This is a good article about the cooling of a Raspberry Pi (in german):

[Raspberry-Pi-4-optimal-kuehlen](https://www.heise.de/ratgeber/Praxistest-Raspberry-Pi-4-optimal-kuehlen-4851547.html)

**Comment by Svardsten53** [comment]
A friend told me about this cooler for the Raspberry Pi 4B. We have tested it on at least three different Pi's and in all cases lowered the temperature by about 20 degrees C. No fan that makes noise. Keep in mind that the aluminum housing shields the wifi signal quite a lot, so cable is preferable or a strong signal. It can be purchased on Amazon.
[Geekworm Raspberry Pi 4 Heavy-duty Aluminum Passive Cooling Metal Case](https://geekworm.com/products/raspberry-pi-4-heavy-duty-aluminum-passive-cooling-metal-case)

**Reply by nilspupils** [reply]
I have both the full armor case you linked to and the open, "half armor" case 
https://geekworm.com/collections/raspberry-pi/products/raspberry-pi-4-model-b-armor-aluminum-alloy-case-protective-shell
and they work fine. Both do get pretty warm in hot weather. I would recommend putting them in a well ventilated area or putting them upright to make the best use of convection. I had no problems with wifi in medium range.

**Comment by AdamM68** [comment]
my rPi 3b+ with two small radiators is running ~60' 
- before, when it was lying on a big stone so it was running at about 53-55'

**Comment by waynecarnold** [comment]
I use this case which has passive cooling and the pi is in my shed. Seems to work well, the highest temp i have seen is about 51 Celsius when it is hot out. Miuzei Raspberry Pi 4 Aluminium Case, Raspberry Pi 4 Model B Metal Case, Pi 4 Alloy Armor Case with Cooling Column and Thermal Pads Compatible with Raspberry pi 4 B 8gb 4gb 2gb- No Pi Board https://a.co/d/7I8KSbL

**Comment by morrowwm** [comment]
I cut a 50mm circle in the lid of my Pi4 case. Then fabricated a conical section from a HDPE food container, and put an 80mm CPU cooler fan on one face of the adapter and hot glued the other face to the case. It's powered off the 5 volt pins on the Pi GPIO header. The fan runs slowly and is quiet. 

[Picture and short description.](https://yeahwellnothing.blogspot.com/2023/06/new-birdnet-case-requires-math.html)

It's about 20C outside right now, and Birdnet sysinfo page reports a CPU temperature of 42C. Headed for low 30s for a high today.
